src/org/parosproxy/paros/extension/option/OptionsViewPanel.java

```diff
@@ -543,6 +568,7 @@ private JCheckBox getScaleImages() {
 		return scaleImages;
 	}
 	
+		

```

Comment:
This white space change could be reverted

https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#formatting